XTS/WS-C/pom.xml

```diff
@@ -116,4 +116,21 @@
       <scope>provided</scope>
     </dependency>
   </dependencies>
+  <profiles>
+    <profile>
+      <id>jdk17</id>
+      <activation>
+        <jdk>[17,)</jdk>
+      </activation>
+      <properties>
+        <jvm.args.modular>
```

Comment:
Generally speaking we do need some of these settings for JDK 17 and not 11 and I am not sure why but not having at least some of these in the change did cause failures. It's been a while since I looked at what was needed but I think it is right to re-review what is needed so I will put it on Hold to reestablish if they are all needed.

The general point of why any are needed in JDK17 though is that there are failures in 17 that are not in 11 when at least some of these are removed.

Reply:
Thanks for the review, I have changed the poms in this area and so am now removing `Hold`. It can be reviewed again.